.github/workflows/build-android.yml

```diff
@@ -118,7 +121,55 @@ jobs:
           ./gradlew build --warning-mode all --stacktrace
 
       - name: Publish
+        id: publish
         run: ./gradlew sdk:publish
         working-directory: crates/bitwarden-uniffi/kotlin
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  update:
+    name: Update sdk test
+    runs-on: ubuntu-24.04
+    needs: combine
+    permissions:
+      contents: read
+      id-token: write
+      packages: write
+
+    steps:
+      - name: Log in to Azure
+        uses: bitwarden/gh-actions/azure-login@main
+        with:
+          subscription_id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+          tenant_id: ${{ secrets.AZURE_TENANT_ID }}
+          client_id: ${{ secrets.AZURE_CLIENT_ID }}
+
+      - name: Get Azure Key Vault secrets
+        id: get-kv-secrets
+        uses: bitwarden/gh-actions/get-keyvault-secrets@main
+        with:
+          keyvault: gh-org-bitwarden
+          secrets: "BW-GHAPP-ID,BW-GHAPP-KEY"
+
+      - name: Log out from Azure
+        uses: bitwarden/gh-actions/azure-logout@main
+
+      - name: Generate GH App token
+        uses: actions/create-github-app-token@0f859bf9e69e887678d5bbfbee594437cb440ffe # v2.1.1
+        id: app-token
+        with:
+          app-id: ${{ steps.get-kv-secrets.outputs.BW-GHAPP-ID }}
+          private-key: ${{ steps.get-kv-secrets.outputs.BW-GHAPP-KEY }}
+          owner: bitwarden
+          repositories: android
+          permission-actions: write
+
+      - name: Trigger SDK update in Android repo
+        if: github.event_name == 'push' || github.event_name == 'workflow_dispatch'
```

Comment:
🤔 We should probably remove `workflow_dispatch` here as it's usually used for feature branch testing and we don't want to open Update PRs on those. 

But we still need an easy way to test the Update process. Perhaps a `workflow_dispatch` input?